spec/features/02_can_sign_in_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+feature 'can sign in with an existing account', :devise do
```

Comment:
Because you're using Devise which is highly tested already, only having a happy path here is fine. That said, I would make sure you're testing the happy and unhappy paths for features in the future, potentially including multiple unhappy edge cases even.